modules/integration/src/test/scala/scala/cli/integration/CompleteTests.scala

```diff
@@ -14,6 +14,15 @@ class CompleteTests extends ScalaCliSuite {
     }
   }
 
+  test("zsh bug") {
+    // guard against https://github.com/alexarchambault/case-app/issues/475
+    TestInputs.empty.fromRoot { root =>
+      val res = os.proc(TestUtil.cli, "complete", "zsh-v1", "2", "scala-cli").call(cwd = root)
```

Comment:
```suggestion
      val res = os.proc(TestUtil.cli, "complete", shellFormat, "2", "scala-cli").call(cwd = root)
```
won't hurt to enforce this to work the same on `bash`.